lib/iris/config.py

```diff
@@ -79,10 +78,7 @@ def get_dir_option(section, option, default=None):
 CONFIG_PATH = os.path.join(ROOT_PATH, "etc")
 
 # Load the optional "site.cfg" file if it exists.
-if sys.version_info >= (3, 2):
-    config = configparser.ConfigParser()
-else:
-    config = configparser.SafeConfigParser()
+config = configparser.ConfigParser()
```

Comment:
The `configparser.SafeConfigParser` class was renamed to `configparser.ConfigParser` in `python 3.2`, see  [here](https://github.com/python/cpython/blob/cd90f6a3692e0f7ef0a13aae651e19a08d1f9b31/Lib/configparser.py#L1224-L1234) for further details.